.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release NPM packages
+
+env:
+  NX_MAX_PARALLEL: 2
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_project:
+        description: 'The target project to release. Defaults to release all affected packages since the last package was released'
+        required: true
+        default: 'affected'
```

Comment:
Verify that we are comparing with HEAD~1 when running affect commands on a merge to main